core/src/main/scala/akka/persistence/cassandra/journal/TagWriter.scala

```diff
@@ -35,33 +36,36 @@ import scala.concurrent.duration._
  */
 @InternalApi private[akka] object TagWriter {
 
-  private[akka] def props(settings: TagWriterSettings, session: TagWritersSession, tag: Tag): Props =
+  @InternalApi private[akka] def props(settings: TagWriterSettings, session: TagWritersSession, tag: Tag): Props =
```

Comment:
if the outer `object` is internal it's not necessary to duplicate that for the inner members

Reply:
Ouch, not sure how I missed that.